lib/policy_machine.rb

```diff
@@ -127,7 +126,9 @@ def is_privilege_ignoring_prohibitions?(user_or_attribute, operation, object_or_
       raise(ArgumentError, "options[:associations] cannot be empty") if associations.empty?
       raise(ArgumentError, "expected each element of options[:associations] to be a PM::Association") unless associations.all?{|a| a.is_a?(PM::Association)}
 
-      associations.keep_if{ |assoc| assoc.includes_operation?(operation) }
+      associations.keep_if do |association|
+        association.operation_set.connected?(operation)
+      end
       return false if associations.empty?
     else
       associations = operation.associations
```

Comment:
note to self: so we filter a collection of associations, check if its empty, and then reassign...wut.